lib/pow/extension/base.ex

```diff
@@ -73,7 +73,11 @@ defmodule Pow.Extension.Base do
         [extension]
         |> Kernel.++(module_list)
         |> Module.concat()
-        |> Code.ensure_compiled?()
+        |> Code.ensure_compiled()
+        |> case do
+          {:module, _} -> true
+          _ -> false
+        end
```

Comment:
I wonder if we should keep this closer to how it [originally worked in in Elixir](https://github.com/elixir-lang/elixir/blob/v1.7.0/lib/elixir/lib/code.ex#L1045-L1047) for `ensure_compiled?`:

```elixir
match?({:module, ^module}, ensure_compiled(module))
```

Though as this is somewhat deprecated logic it doesn't really matter 😄

Reply:
Seems to me that match?/2 isn't deprecated, so I don't see why not.

Made changes.